include/jemalloc/internal/quantum.h

```diff
@@ -71,6 +71,9 @@
 #  ifdef __arc__
 #    define LG_QUANTUM		3
 #  endif
+#  if (defined(__loongarch64) || defined(__loongarch__))
```

Comment:
Let's update the existing branch here:

https://github.com/jemalloc/jemalloc/blob/dev/include/jemalloc/internal/quantum.h#L33